app/screens/post_options/post_option.js

```diff
@@ -41,12 +43,28 @@ export default class PostOption extends PureComponent {
         const {destructive, icon, onPress, text} = this.props;
         const image = icons[icon];
 
+        const Touchable = Platform.select({
+            ios: TouchableHighlight,
+            android: TouchableNativeFeedback,
+        });
+
+        const touchableProps = Platform.select({
+            ios: {
+                underlayColor: 'rgba(0, 0, 0, 0.05)',
+            },
+            android: {
+                background: TouchableNativeFeedback.Ripple( //eslint-disable-line new-cap
```

Comment:
Good job, Facebook. Way to just ignore standard JS conventions for one random component 😛